modules/reference/pages/config/server-configuration-overview.adoc

```diff
@@ -194,6 +195,23 @@ For example, if you enter `${my.env.var}` in the `server.xml` file, it looks for
 . my_env_var
 . MY_ENV_VAR
 
+Variables can be loaded from files that are located in directories that are specified by the `VARIABLE_SOURCE_DIRS` environment variable.
```

Comment:
`The name of the file is used as the variable's name..`

Per IBM style, don't use possessive (`s) for inanimate objects. In this case I think you can just drop the `s:

`The name of the file is used as the variable name..`